### DISCUSSION TITLE
The Lord of the Rings: Gollum UE 4.27.2 Support

### DISCUSSION THREAD

**Discussion by GhostHeroine** [gildor2/UEViewer#301] [Q&A] [answered]
Hi! UE Viewer can see what is inside .pak files of The Lord of the Rings: Gollum, however, it cannot export  any files at all... Seems UE version here for that game is 4.27.2 as mention in .exe file properties. What do you think? If this possible to made an update of program to make it work with that game? Thanks

**Comment by gildor2** [comment, marked as answer]
https://www.gildor.org/smf/index.php/topic,8590.0.html
